# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/cvm/head1.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile	2003-11-18 00:50:48.000000000 +0100
+++ /tmp/tmp.awWCZS5786	2004-05-24 13:23:16.000000000 +0200
@@ -33,16 +33,16 @@
 	./makelib command.a module_command.o module_command_main.o module_request.o module_output.o facts.o
 
 compile: conf-cc conf-bgincs
-	( bgincs=`head -1 conf-bgincs`; \
+	( bgincs=`head -n 1 conf-bgincs`; \
 	  echo '#!/bin/sh'; \
 	  echo 'source=$$1; shift'; \
 	  echo 'base=`echo "$$source" | sed -e s:\\\\.c$$::`'; \
-	  echo exec `head -1 conf-cc` -I. "-I'$${bgincs}'" '-o $${base}.o -c $$source $${1+"$$@"}'; \
+	  echo exec `head -n 1 conf-cc` -I. "-I'$${bgincs}'" '-o $${base}.o -c $$source $${1+"$$@"}'; \
 	) >compile
 	chmod 755 compile
 
 conf_home.c: conf-home
-	head -1 conf-home | \
+	head -n 1 conf-home | \
 	  sed -e 's/"/\\"/g' \
 	      -e 's/^/const char conf_home[] = "/' \
 	      -e 's/$$/";/' >conf_home.c
@@ -106,10 +106,10 @@
 libraries: client.a udp.a local.a command.a module.a sasl.a
 
 load: conf-ld conf-bglibs
-	( bglibs=`head -1 conf-bglibs`; \
+	( bglibs=`head -n 1 conf-bglibs`; \
 	  echo '#!/bin/sh';\
 	  echo 'main="$$1"; shift';\
-	  echo exec `head -1 conf-ld` -L. "-L'$${bglibs}'" '-o "$$main" "$$main.o" $${1+"$$@"}' -lbg-sysdeps; \
+	  echo exec `head -n 1 conf-ld` -L. "-L'$${bglibs}'" '-o "$$main" "$$main.o" $${1+"$$@"}' -lbg-sysdeps; \
 	) >load
 	chmod 755 load
 
